zstacklib/zstacklib/utils/lvm.py

```
@@ -649,36 +649,130 @@ def config_lvm_by_sed(keyword, entry, files):
+_ACTIVE_GLOBAL_FILTER_RE = re.compile(r'^\s*global_filter(?:\s|=)')
+_COMMENT_RE = re.compile(r'^\s*#')
+
+
+def _build_filter_value(preserve_disks=None, no_drbd=False, vgs=None):
+    """Build the bracketed filter value string, e.g. '["a|^/dev/sda$|", "r|.*|"]'.
+
+    Caller decides whether to put it on a 'filter = ...' or 'global_filter = ...' line.
+    """
+    if preserve_disks is not None and len(preserve_disks) != 0:
+        accept = ['"a|^%s$|"' % disk for disk in preserve_disks]
+        return '[' + ', '.join(accept + ['"r|.*|"']) + ']'
```

Ruff (0.15.12)

[warning] Consider `[*accept, '"r|.*|"']` instead of concatenation

Replace with `[*accept, '"r|.*|"']`

(RUF005)